### DISCUSSION TITLE
How to set this animation for popupmenu in WPF?

### DISCUSSION THREAD

**Discussion by VahidEra** [dotnet/wpf#7847] [Q&A]
Hi guys,
I have a popupbox in my project and I want it to have this animation when it opens:
![1](https://github.com/dotnet/wpf/assets/27216710/65f0d2bd-ff09-4be6-87bd-bf2cfe0dd4bf)

**_I slowed the movie down so its animation is better visible:_**

![2](https://github.com/dotnet/wpf/assets/27216710/002951ee-78e0-43f9-825b-4d5008c3f42e)

I think two transparency and scale options have been used for this animation, the scale is created step by step and the value has not been suddenly transferred from 0 to 1, of course this is my guess.
I tried to produce this animation with the help of my dear friend @miloush, but I did not get the necessary result Because I only noticed the animation movements by slowing down the opening of the popup box. And I ask you to help me in completing this animation if you can.  For your convenience, I have provided the project file below.  Thanks for your help.

[ **This is sample project**](https://drive.google.com/file/d/1R7mAYonUWItowRLOOlhy-A95xExcJ56M/view?usp=sharing)

**Comment by miloush** [comment]
If you are using controls of a 3rd party library I think you should turn to them for support. I don't see how you can achieve this with PopupBox, but I am no expert on the library.

**Reply by VahidEra** [reply]
@miloush Dear Jan, yes, I got it.  Thank you for your guidance.
